src/gui/editors/SongEditor.cpp

```diff
@@ -76,11 +76,14 @@ SongEditor::SongEditor( Song * song ) :
 	TrackContainerView( song ),
 	m_song( song ),
 	m_zoomingModel(new ComboBoxModel()),
+	m_snappingModel(new ComboBoxModel()),
+	m_proportionalSnap( true ),
```

Comment:
Since you disabled proportional snap by default, this should be `false`.

Reply:
Good catch. I'll hold off on changing this until we resolve your comment about displaying the "real" snap size.